windows/src/global/vc/kmtip_guids.cpp

```diff
@@ -0,0 +1,4 @@
+#include "pch.h"
+
+const CLSID c_clsidKMTipTextService = {0xFE0420F1, 0x38D1, 0x4B4C, {0x96, 0xBF, 0xE7, 0xE2, 0x0A, 0x74, 0xCF, 0xB7}};
+const char cs_clsidKMTipTextService[] = "FE0420F1-38D1-4B4C-96BF-E7E20A74CFB7";
```

Comment:
```suggestion
const char cs_clsidKMTipTextService[] = "{FE0420F1-38D1-4B4C-96BF-E7E20A74CFB7}";
```

Given the following from [StringFromGUID2 docs](https://docs.microsoft.com/en-us/windows/win32/api/combaseapi/nf-combaseapi-stringfromguid2):

> `[out] lpsz`: A pointer to a caller-allocated string variable to receive the resulting string. The string that represents rguid includes enclosing braces.

Reply:
hmm I check the logging on this.  I guess it is working because it is still a subset. with or without the brackets.